### DISCUSSION TITLE
[Question] There is a new milestone for TheXTech?

### DISCUSSION THREAD

**Discussion by agergo100** [Wohlstand/TheXTech#337] [Q&A]
I like the way to playing SMBX on android, but I interested about what new kind of features should be added.

**Comment by Wohlstand** [comment]
The Android support is already here since a while ago including current stable release. You will need:
- install engine's apk
- download one of assets packs (one pack represents the full resources directory of SMBX with all default graphics, music, sounds, and episodes bundled)
- unpack the assets pack into any convenient directory (preferred to be on phone's storage, but can be put on SD card, but gamesaves won't written on SD because of some problems, they will be saved at app data directory)
- when run game, look at settings and select the directory with assets, and then you'll be able to launch the game

However, to simplify life of others, I still want to write a short article on how to install all the stuff.

**Comment by agergo100** [comment]
Are you planting a new feature?

**Comment by Wohlstand** [comment]
Speaking about new features, you already can look at what new was added to currently in-development version here:
<https://github.com/Wohlstand/TheXTech/blob/master/changelog.txt> and all planning features were listed at issues here and assigned to specific milestones. The next planned version is 1.3.6.

**Comment by Wohlstand** [comment]
btw, Questions are reasonable to post at Discussions then, lemme me move this...
